src/util/mod.rs

```diff
@@ -122,3 +123,10 @@ impl<T> HandyRwLock<T> for RwLock<T> {
         self.read().unwrap()
     }
 }
+
+
+pub fn make_std_tcp_conn<A: ToSocketAddrs>(addr: A) -> Result<TcpStream, Error> {
```

Comment:
Actually, we are still using still mio::tcp::TcpStream in server module, so i have to use std_tcp to separate them.